src/plib/resources/locales/en-US.php

```diff
@@ -51,6 +55,9 @@
     'promo.textDone' => 'Plesk has been secured!',
     'promo.buttonSecure' => 'Secure it!',
     'promo.buttonDone' => 'Check it out',
+
+    'controllers.symantec.please-install' => 'Please <a href="javascript:Jsw.redirectPost(\'%%url%%\')">install the Symantec SSL extension</a>',
```

Comment:
locale is not a good place for code: javascript:Jsw.redirectPost should be part of view